File: SCRIPTS/2.1/escolha_maior_menor.js
var number = Math.floor(Math.random() * 10)
var start = true

while (start) {
     user_number = prompt('Chute um numero:: ')
     if (user_number > number) {
         console.log('Menor!')
     }
     else if (user_number < number) {
         console.log('Maior!')
     }
     else if (user_number == number) {
         console.log('Você ganhou!')
         break
     }
}

File: SCRIPTS/2.1/escolha_maior_menor.py
import random

number = random.randint(1,10)
start = True

while start:
    user_number = int(input('Escolha um numero:: '))
    if user_number > number:
        print('Menor!')
    elif user_number < number:
        print('Maior!')
    elif user_number == number:
        print('Você ganhou!!!')
        break